addons/mail/views/mail_channel_member_views.xml

```diff
@@ -6,7 +6,7 @@
         <field name="model">mail.channel.member</field>
         <field name="priority">10</field>
         <field name="arch" type="xml">
-            <tree string="Channels">
+            <tree string="Channels" create="0">
```

Comment:
I would revert this diff part for now, I don't really see the need to change that in stable (it doesn't fix the bug).

In master, what could be done is to allow actually setting a value for channel_id when creating a new record, and only making it readonly when it is already set (+ adding sql constraint).

Reply:
@seb-odoo thanks, should i remove from both the tree views or only this tree view ?

Reply:
Just keep the change in the compute in this PR.

Reply:
@seb-odoo updated the pr